NEW_RELEASE_NOTES.md

```diff
@@ -7,3 +7,4 @@ for next branch cut* header.
 appropriate header in [RELEASE_NOTES.md](./RELEASE_NOTES.md).
 
 ## Release notes for next branch cut
+- backend: fix #6997 : picking can fail on Adreno [⚠️ **New Material Version**]
```

Comment:
Does this _require_ a material bump, or only if users want the fix?

Reply:
I think it requires it.